ipaserver/install/server/replicainstall.py

```diff
@@ -659,7 +655,12 @@ def install_check(installer):
     filename = installer.replica_file
     installer._enrollment_performed = False
 
-    # check FIPS, selinux status, http and DS ports, NTP conflicting services
```

Comment:
Please remove the FIPS comment from [L#952](https://github.com/freeipa/freeipa/blob/master/ipaserver/install/server/replicainstall.py#L952) as well.